### PR TITLE
[netcore] [51.cafe-bot] Add missing QnA resource files

### PR DESCRIPTION
**Pull request by VictorGrycuk**
Fixes #748

## Proposed Changes
Add missing QnA resource files required to generate LUIS models, as mentioned in the [readme](https://github.com/Microsoft/BotBuilder-Samples/tree/06dfb1b074563ef390b837535593f5e45e76eac1/samples/csharp_dotnetcore/51.cafe-bot#building-and-creating-services).

## Testing
Running the following command will not throw an error anymore:
```bash
ludown parse toluis --in Dialogs/Dispatcher/Resources/cafeDispatchModel.lu -o cognitiveModels -n cafeDispatchModel --out cafeDispatchModel.luis
```

### CODE REVIEW

**Review by joseques — APPROVED** [review]
:shipit:

File: samples/csharp_dotnetcore/51.cafe-bot/Dialogs/QnA/resources/qna.lu [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+## ? high chair?
+- do you have high chair for kids? 
+- can i bring my kids?
+- do you have high chairs?
+- is it ok to bring my son?
+- can i bring my daughter along?
+```markdown
+Guests all age are welcome at Contoso Cafe! We offer convenient seating for all age groups and can provide high chairs if needed. We are more than happy to accomodate your needs while you are here. Just let one of the staff know!
```
Comment (dfavretto) [inline samples/csharp_dotnetcore/51.cafe-bot/Dialogs/QnA/resources/qna.lu]: There's a typo here with `accomodate`, it should be `accommodate` with double `m`.

File: samples/csharp_dotnetcore/51.cafe-bot/Dialogs/QnA/resources/qna.lu [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- grandson
+
+## ? what's your layout?
+- so what's your store's layout? 
+- typical layout of your coffee shops
+- what is your store layout
+- what is the layout of your stores?
+- does 
+```markdown
+Sure.  We have general seating area that can accomodate up to 50 guests at a time. We also feature comfortable coaches, chairs, tables and barstools in our cafe.  Oh and we have electrical outlets just about everywhere we could possible shove them.
```
Comment (dfavretto) [inline samples/csharp_dotnetcore/51.cafe-bot/Dialogs/QnA/resources/qna.lu]: Same as before, typo in `accomodate`

**Review by dfavretto — CHANGES_REQUESTED** [review]

File: samples/csharp_dotnetcore/51.cafe-bot/Dialogs/QnA/resources/qna.lu (hunk 2 above)
Comment (VictorGrycuk) [reply samples/csharp_dotnetcore/51.cafe-bot/Dialogs/QnA/resources/qna.lu]: fixed

File: samples/csharp_dotnetcore/51.cafe-bot/Dialogs/QnA/resources/qna.lu (hunk 1 above)
Comment (VictorGrycuk) [reply samples/csharp_dotnetcore/51.cafe-bot/Dialogs/QnA/resources/qna.lu]: fixed

**Review by dfavretto — APPROVED** [review]

**Comment by dfavretto** [comment]
Closed due to Microsoft#785

**Pull request closed without merging**